test/Concurrency/Runtime/async_task_locals_copy_to_sync.swift

```diff
@@ -76,8 +76,11 @@ func copyTo_sync_noValues() {
 /// Similar to tests in `async_task_locals_copy_to_async_ but without any task involved at the top level.
 @available(SwiftStdlib 5.5, *)
 @main struct Main {
-  static func main() {
-    copyTo_sync_noWait()
-    copyTo_sync_noValues()
+  static func main() async {
+    // wrap with Task to insert unstructured tasks before the end of main job
+    await Task {
+      copyTo_sync_noWait()
+      copyTo_sync_noValues()
+    }.get()
```

Comment:
Per the comment above in the source, the point of this test is that it does not run inside a task, because it's checking that the task-locals code works when there is no top-level task. If this can't be made to work with the cooperative executor, then we should disable the test.

Reply:
I see. I disabled this case for the cooperative case.